components/JsGeoda.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useEffect } from "react";
+import { useSelector } from "react-redux";
+import { GeoJsonLayer } from "@deck.gl/layers";
+import { StaticMap } from "react-map-gl";
+import colorbrewer from "colorbrewer";
+import jsgeoda from "jsgeoda";
+import DeckGL from '@deck.gl/react';
+
+
+const MAPBOX_TOKEN = process.env.NEXT_PUBLIC_MAPBOX_TOKEN;
+
+const INITIAL_VIEW_STATE = {
+  longitude: -100.4,
+  latitude: 38.74,
+  zoom: 2.5,
+  maxZoom: 20
+};
+
+const JsGeoda = () => {
+  const [layer, setLayer] = useState(null);
+  const fileData = useSelector(state => state.root.file.fileData);
+
+  useEffect(() => {
+    const loadSpatialData = async (geoda) => {
+      const nat = geoda.readGeoJSON(fileData);
```

Comment:
This is one of the refactors we want to do for jsgeoda: accept a GeoJSON object parsed by loaders.gl instead of an arraybuffer